app/src/main/java/com/omelan/cofi/MainActivity.kt

```diff
@@ -204,9 +207,17 @@ class MainActivity : MonetCompatActivity() {
                                 it.copy(recipeId = idOfRecipe.toInt())
                             }
                         )
+                                    isRecipeSave=true
+                    }else{Toast.makeText(applicationContext,"Name and description can't be empty.",Toast.LENGTH_SHORT).show()  }
```

Comment:
There shouldn't be any hardcoded strings. All strings should be in `values/strings.xml` and refereed in code via resource id